src/Fridge.java

```diff
@@ -0,0 +1,57 @@
+import components.map.Map;
+import components.map.Map1L;
+
+public class Fridge {
+    private boolean isOpen;
+    private static Map<String, Integer> fridgeItems;
+    private Map<String, Integer> freezerItems;
+
+    public Fridge() {
+        this.isOpen = false;
+        this.fridgeItems = new Map1L<>();
+        this.freezerItems = new Map1L<>();
+    }
+
+    public void addOneItem(String item, boolean isOpen,
+            Map<String, Integer> location) {
+        if (!isOpen) {
+            if (location.hasKey(item)) {
+                location.replaceValue(item, location.value(item) + 1);
+            } else {
+                location.add(item, 1);
+            }
+        }
+    }
+
+    public void removeOneItem(String item, boolean isOpen,
+            Map<String, Integer> location) {
+        if (!isOpen) {
+            location.remove(item);
+        }
+    }
+
+    public boolean inFridge(String item) {
+        return this.fridgeItems.hasKey(item);
+    }
+
+    public boolean inFreezer(String item) {
+        return this.freezerItems.hasKey(item);
+    }
+
+    public int numOfItem(String item) {
```

Comment:
Actually, methods generally perform actions, so we usually start them with a verb:

```suggestion
    public int getNumOfItem(String item) {
```

I know that isn't true for the OSU API. For instance, we have `size` and `root` methods, but I tend to prefer verbs for methods in general.